Error when building on python 3.14.

As the title.

The error is:

```
Error: cibuildwheel: Command ['/usr/bin/arch', '-x86_64', 'pip', 'install', '/private/var/folders/bp/kmfmhnl95kx1c8x321z7twbw0000gn/T/cibw-run-jmqq_blg/cp314-macosx_x86_64/repaired_wheel/ansys_mapdl_reader-0.55.2-cp314-cp314-macosx_10_15_x86_64.whl'] failed with code 1. 

Error: Process completed with exit code 1.
```

Reference: https://github.com/ansys/pymapdl-reader/actions/runs/20656889258/job/59311375746#step:3:5180

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -132,7 +132,7 @@ jobs:
           retention-days: 7
 
   mac_build:
-    # if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
     name: Build wheels on MacOS
     runs-on: macos-latest
 

